fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -3089,7 +3089,10 @@ private boolean areAllRecordTypesSince(@Nullable Collection<RecordType> recordTy
             }
         }
 
-        return markIndexReadable(index).thenApply(b -> null);
+        return markIndexReadable(index).thenApply(vignore -> {
```

Comment:
Possibly either `.thenRun` or `.thenAccept` instead of `.thenApply`. It doesn't really matter, though; just style.

Reply:
Done.